### PR TITLE
PHP notice fixes on options page

### PR DESCRIPTION
**Pull request by colemanw**
Overview
----------------------------------------
Fixes php notices, short array syntax, and a bug-begging typo on the Options listing page.

Before
----------------------------------------
Go to e.g. "Website Types".
Notice the notices.

After
----------------------------------------
No notices to notice.

### CODE REVIEW

File: CRM/Admin/Page/Options.php [hunk 1]
```diff
@@ -229,7 +226,7 @@ public function browse() {
     );
 
     // retrieve financial account name for the payment method page
-    if ($gName = "payment_instrument") {
+    if ($gName === "payment_instrument") {
```
Comment (colemanw) [inline CRM/Admin/Page/Options.php:229]: Yikes. This was a typo that caused the condition to always be true!

File: templates/CRM/Admin/Page/Options.tpl [hunk 2]
```diff
@@ -137,7 +137,7 @@
             <td class="crm-admin-options-description crm-editable" data-field="description" data-type="textarea">{if isset($row.description)}{$row.description}{/if}</td>
             <td class="nowrap crm-admin-options-order">{if isset($row.weight)}{$row.weight}{/if}</td>
             {if !empty($showIsDefault)}
-              <td class="crm-admin-options-is_default" align="center">{icon condition=$row.is_default}{ts}Default{/ts}{/icon}&nbsp;</td>
+              <td class="crm-admin-options-is_default" align="center">{if !empty($row.is_default)}{icon}{ts}Default{/ts}{/icon}{/if}&nbsp;</td>
```
Comment (colemanw) [inline templates/CRM/Admin/Page/Options.tpl:140]: The `condition=$row.is_default` caused notices, and it's the same as surrounding the whole block in `{if}`

**Comment by monishdeb** [comment]
Jenkins test this please

**Comment by monishdeb** [comment]
Unrelated failure but still good to have a clean build :)

**Comment by seamuslee001** [comment]
Jenkins test this please

**Pull request merged**